requirements.txt

```diff
@@ -5,7 +5,7 @@ requests
 networkx >= 2.0.0
 
 # rigetti packages
-rpcq >= 2.7.2
+rpcq == 3.0.0
```

Comment:
`==` or `>=`?

Reply:
haha - there's an ongoing discussion about that, I was recently corrected from `>=` to `==` for `rpcq` on a different project, so I'll defer to @karalekas

Reply:
Ah. Fair enough. I'm fine with either. I just noticed it had changed and wanted to make sure it was intentional.

Reply:
`>=`; also will need to be updated in the `setup.py`